internal/handlers/irc/irc.go

```diff
@@ -17,11 +18,20 @@ type Client struct {
 	sendToTg func(string)
 }
 
+/*
+Creating variables for logging
+*/
+var (
+	logFlags    = log.Ldate | log.Ltime | log.Lmicroseconds | log.Llongfile
```

Comment:
Could you please switch instances of `Llongfile` to `Lshortfile`? I think having the fully qualified name is most unnecessary here, since most deployments will use something like systemd or containers and the full path wouldn't be very helpful in these situatoins.